corehq/apps/reports/standard/forms/reports.py

```diff
@@ -58,7 +59,7 @@ def submitfilter(self):
 
     @property
     def sort_params(self):
-        sort_col_idx = int(self.request.GET.get('iSortCol_0', None))
+        sort_col_idx = int(self.request.GET['iSortCol_0'])
```

Comment:
If the default ever fired the outer `int` would raise a `ValueError` so I figured we could just make the fact that it's required explicit.